### ISSUE TITLE
Structure

### ISSUE THREAD

**Issue by ljstrnadiii** [adri-romsor/DietNetworks#3]
Howdy,

It's less of an issue and more for clarity. I was wondering by default what is the structure of the model that is successful. It seems like:
Params: (no dropout, yes batch norm, .001 learning rate, mostly tanh activations except the final for reconstruction,  batch size of 128, uniform init w std=.02 for all weights, no regularizers on weights, no weight clipping)

With a structure:
2 hidden layers for post histogram embedding before passing learned weights to 2 hidden layers with inputs to softmax and reconstruction using 2 hidden layers with embedding before passing to mse...

It's hard to fully describe in a compact way is there anywhere this exact structure was described? The paper does not go into the details of what you guys had to do to make it learn. 

I am rebuilding this in tensorflow and I can't seem to get the training error above 50%

Thanks for any help
